bentoml/handlers/dataframe_handler.py

```diff
@@ -66,6 +66,36 @@ def __init__(
         self.typ = typ
         self.input_columns = input_columns
 
+    def _get_type(self, item):
+        if item.startswith('int'):
+            return 'integer'
+        if item.startswith('float') or item.startswith('double'):
+            return 'number'
+        if item.startswith('str'):
+            return 'string'
+        return 'object'
+
+    @property
+    def request_schema(self):
+        default = {"application/json": {"schema": {"type": "object"}}}
+        if self.input_columns is None:
+            return default
+
+        if isinstance(self.input_columns, dict):
```

Comment:
Might be good to support list as input as well?

for example in the quick start, if you look at the input dtypes:
```pd.DataFrame(X).dtypes.to_dict()```

the columns names is basically the index:
```{0: dtype('float64'),
 1: dtype('float64'),
 2: dtype('float64'),
 3: dtype('float64')}
```

In those cases it might be easier to simply do:
```@api(DataframeHandler, input=['float', 'float', 'float', 'float'])```

and API client will send similar json format to the API endpoint:
```
curl -i \
--header "Content-Type: application/json" \
--request POST \
--data '[[5.1, 3.5, 1.4, 0.2]]' \
localhost:5000/predict
```

Reply:
Hi, this is cool, but I haven't found a solution to implement this in a less complicated way.
Can I extend this in next PRs?

Also, if we support list type, the `check_dataframe_column_contains` needs to refactor too, as pointed out by @yubozhao

Reply:
yea sounds great, let's do it and refactor check_dataframe_column_contains in follow up PR

Reply:
btw, I was thinking of a simple conversion here when seeing an array:
```
if isinstance(input_dtypes, list):
  input_dtypes = dict((index, dtype) for index, dtype in enumerate(ic))
```
pd.Dataframe.read_json should already handling the reading part, both way works fine

Reply:
Good idea.

The question here is I'm not sure how it works on Open API that we support two formats for input parameters, ie, one is `[[1, 2]]`, another is `{"col1":[1],"col2":[2]}`.
Basically no matter user put a list or a dict into `input_dtypes`, we need to support the above 2 formats on API docs. Is my understanding right?